src/Domain/GetSongsList.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+declare(strict_types=1);
+
+function getSongs()
+{
+    $dataList=file_get_contents('../../repository/songs_list.json');

```

Comment:
Please make sure you add one space before and after the equals sign (https://www.php-fig.org/psr/psr-12/#62-binary-operators).